test/sql/test_delete.py

```diff
@@ -77,12 +78,14 @@ def test_delete(self):
     def test_where_empty(self):
         table1 = self.tables.mytable
 
-        self.assert_compile(
-            table1.delete().where(and_()), "DELETE FROM mytable"
-        )
-        self.assert_compile(
-            table1.delete().where(or_()), "DELETE FROM mytable"
-        )
+        with expect_deprecated():
```

Comment:
**mike bayer** (zzzeek) wrote:

use boolean_empty here, but also add new tests for the actual "empty" version in test/sql/test_deprecations.py